bitnami/mongodb-sharded/templates/mongos/mongos-service.yaml

```diff
@@ -44,6 +44,11 @@ spec:
     - name: metrics
       port: 9216
       targetPort: metrics
+      {{- if .Values.service.nodePorts.metrics }}
```

Comment:
Please add this new value (`service.nodePorts.metrics`) and its documentation to the `values.yaml` file: https://github.com/bitnami/charts/blob/fe92d1b2ed3277e8c8549f5732544253ea1178ba/bitnami/mongodb-sharded/values.yaml#L316-L320